packages/devtools/src/commands/isElementDisplayed.js

```diff
@@ -1,11 +1,24 @@
+import executeScript from './executeScript'
+
 import { getStaleElementError } from '../utils'
+import { ELEMENT_KEY } from '../constants'
+
+const SCROLL_INTO_VIEW_SCRIPT = 'return (function scrollIntoView (elem) { elem.scrollIntoView() }).apply(null, arguments)'
```

Comment:
What do you think on changing scrollIntoView to scrollIntoViewIfNeeded?

Reply:
However scrollIntoViewIfNeeded command is non standard.
Ok then, can we get element visibility, if it is not visible then scroll to it and try again?

So my complain is that I won't be able to use `isDisplayed` command anymore with current implementation :(

Reply:
> So my complain is that I won't be able to use isDisplayed command anymore with current implementation :(

Why is that?

Reply:
https://github.com/webdriverio/webdriverio/pull/4459/files#r321205421

> Open https://go.moplay.com/de/sports/registration-pop-up/?Offer=Bet10Get30GenSports
wait for site to load and form to open, then do in browser
$$('[data-step="1"] .stepwise-registration__ssl')[0].scrollIntoView()
That's it, form is broken :(

So I'm not able to use isDisplayed in forms like that